src/app/beer_garden/scheduler.py

```diff
@@ -523,7 +523,8 @@ def run_job(job_id, request_template, **kwargs):
         if updates != {}:
             db.modify(db_job, **updates)
     except Exception as ex:
-        logger.exception(f"Error executing {db_job}: {ex}")
+        logger.error(f"Error executing {db_job}: {ex}")
+        db.modify(db_job, {"inc__error_count": 1})
```

Comment:
This line needs to be:

```python
db.modify(db_job, inc__error_count=1)
```

`modify` is expecting to deal with kwargs.  As is this just ends up raising another exception rather than incrementing the error counter.

Reply:
Adding a double start is equivalent, but your way is more straightforward and therefore preferable.